warehouse/models/intermediate/gtfs_quality/guidelines_checks/int_gtfs_quality__schedule_download_success.sql

```diff
@@ -10,27 +12,29 @@ fct_schedule_feed_downloads AS (
 
 daily_feed_download_unzip_success AS (
     SELECT date,
-           base64_url
+           base64_url,
+           LOGICAL_OR(download_success) AS download_success,
+           LOGICAL_OR(unzip_success) AS unzip_success
       FROM fct_schedule_feed_downloads
-     WHERE download_success
-       AND unzip_success
+      GROUP BY 1, 2
 ),
 
-int_gtfs_quality__schedule_download_success AS (
+int_gtfs_quality__schedule_url_download_success AS (
     SELECT
-        idx.date,
-        idx.base64_url,
-        {{ static_feed_downloaded_successfully() }} AS check,
-        {{ compliance_schedule() }} AS feature,
+        idx.* EXCEPT(status),
         CASE
-            WHEN LOGICAL_AND(d.base64_url IS NOT null) THEN {{ guidelines_pass_status() }}
-            ELSE {{ guidelines_fail_status() }}
+            WHEN download_success AND unzip_success THEN {{ guidelines_pass_status() }}
+            -- means we attempted to download and failed
+            WHEN idx.has_schedule_url AND d.base64_url IS NOT NULL THEN {{ guidelines_fail_status() }}
+            -- means we did not attempt to download on this day for whatever reason
+            WHEN idx.has_schedule_url AND d.base64_url IS NULL THEN {{ guidelines_na_check_status() }}
+            -- fill in n/a entities etc.
```

Comment:
is this TODO item still supposed to be here?

Reply:
Ope sorry I think that's not a TODO but just a description of what the `ELSE` condition is doing. I can clarify this. And I think I'm gonna add more comments in the `CASE` statements in the other checks too because they are kind of a doozy.... As alluded in a Slack message, I think that I am trading off here for a large number of relatively simple models that have to be brought together in a complex way for a smaller number of more complex models but where the combination logic is therefore not as complicated

Reply:
ah thanks for clarifying. i'm still getting used to the new model, i thought you were saying "i still need to fill in the n/a entities".